task16/index.js

```diff
@@ -261,18 +83,74 @@ const renderTabContainer = (value) => {
 const createArticleTitle = ({ article }) => {
```

Comment:
IMO
Why not make it an `articleItem` to match the class name?
```suggestion
const createArticleItem = ({ article }) => {
```

Reply:
Here, too, the function name did not match the content. I had not noticed this until you pointed it out. I will take care.
e44ebc6